test/Interop/SwiftToC/functions/swift-primitive-functions-c-bridging.swift

```diff
@@ -4,7 +4,7 @@
 
 // RUN: %check-interop-c-header-in-clang(%t/functions.h)
 
-// CHECK: SWIFT_EXTERN float $s9Functions16passThrougCFloatyS2fF(float x) SWIFT_NOEXCEPT SWIFT_CALL;
+// CHECK: SWIFT_EXTERN float $s9Functions16passThroughCFloatyS2fF(float x) SWIFT_NOEXCEPT SWIFT_CALL;
```

Comment:
```suggestion
// CHECK: SWIFT_EXTERN float $s9Functions17passThroughCFloatyS2fF(float x) SWIFT_NOEXCEPT SWIFT_CALL;
```

Reply:
The other occurrences of this mangled name need an update too.

Reply:
<img width="1546" alt="Screenshot 2024-08-09 at 07 42 40" src="https://github.com/user-attachments/assets/e0e60054-69bd-45fa-b533-a8a4b148ad6f">

ops, I had missed that comment, thx! Find replace all found these, which I've included in the amended commit.